parallel-install/pkg/deployment/deletion.go

```diff
@@ -313,120 +282,53 @@ func (i *Deletion) deleteKymaNamespaces(namespaces []string) error {
 					i.cfg.Log.Infof("Deleted finalizer from Secret: %s", secret.Name)
 				}
 
-				//HACK: Delete finalizers of leftover ORY Rules
-				ruleResource := schema.GroupVersionResource{
-					Group:    "oathkeeper.ory.sh",
-					Version:  "v1alpha1",
-					Resource: "rules",
-				}
-
-				rules, err := i.dClient.Resource(ruleResource).Namespace(ns).List(context.Background(), metav1.ListOptions{})
+				//HACK: Delete finalizers of leftover Custom Resources
```

Comment:
If I understand this correctly we're only removing finalizers on Custom Resources and one specific secret, are we good with not checking core resources for example?